Documentation/BuildInstructions.md

```diff
@@ -74,46 +78,53 @@ Notes:
 - bash is needed because the default version installed on macOS doesn't support globstar
 - If you install some commercial EXT2 macOS fs handler instead of osxfuse and fuse-ext2, you will need to `brew install e2fsprogs` to obtain `mke2fs` anyway.
 - As of 2020-08-06, you might need to tell the build system about your newer host compiler. Once you've built the toolchain, navigate to `Build/`, `rm -rf *`, then run `cmake .. -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10`, then continue with `make install` as usual.
+- You may also want to install [ninja](https://github.com/ninja-build/ninja/releases)
 
 #### OpenBSD prerequisites
 ```
 pkg_add bash gmp gcc git gmake sudo
 ```
 
+<!-- FIXME: Replace gmake with ninja -->
+
 #### FreeBSD prerequisites
 ```
 $ pkg add coreutils gmake bash sudo git
 ```
 
+<!-- FIXME: Replace gmake with ninja -->
+
 #### Windows
 For Windows, you will require Windows Subsystem for Linux 2 (WSL2). [Follow the WSL2 instructions here.](https://github.com/SerenityOS/serenity/blob/master/Documentation/NotesOnWSL.md)
 Do note the ```Hardware acceleration``` and ```Note on filesystems``` sections, otherwise performance will be terrible.
 Once you have installed a distro for WSL2, follow the Linux prerequisites above for the distro you installed, then continue as normal.
 
+You may also want to install [ninja](https://github.com/ninja-build/ninja/releases)
+
 ### Build
 Go into the `Toolchain/` directory and run the **BuildIt.sh** script:
 ```bash
 $ cd Toolchain
 $ ./BuildIt.sh
 ```
 
-Building the toolchain will also automatically create a `Build/` directory for the build to live in, and build cmake inside that directory.
+Building the toolchain will also automatically create a `Build/` directory for the build to live in.
 
-Once the toolchain and cmake have been built, go into the `Build/` directory and run the `make` and `make install` commands:
+Once the toolchain has been built, go into the `Build/` directory and run the commands. Note that `ninja` seems to be faster, you can also just use GNU make, by omitting `-g Ninja` and calling `make` instead of `ninja`:
```

Comment:
the g in `-g Ninja` should be uppercase, right?

Reply:
It's already correct, isn't it?

Reply:
no, it's using lowercase ``g``. Forgive me if I'm wrong.

Reply:
Ah, now I see! I was too focused on `Ninja` itself. Yes, I'll change that!